test/c3/appframework/appframework_test.go

```diff
@@ -106,6 +108,46 @@ var _ = Describe("c3appfw test", func() {
 			initContDownloadLocation := "/init-apps/" + appSourceName
 			podNames := []string{fmt.Sprintf(testenv.ClusterMasterPod, deployment.GetName()), fmt.Sprintf(testenv.DeployerPod, deployment.GetName())}
 			testenv.VerifyAppsDownloadedByInitContainer(deployment, testenvInstance, testenvInstance.GetName(), podNames, appListV1, initContDownloadLocation)
+
+			//Verify Apps are copied to location
+			allPodNames := testenv.DumpGetPods(testenvInstance.GetName())
+			testenv.VerifyAppsCopied(deployment, testenvInstance, testenvInstance.GetName(), allPodNames, appListV1, true, true)
+
+			//Verify Apps are installed
+			for _, podName := range allPodNames {
+				if strings.Contains(podName, "-indexer-") || strings.Contains(podName, "-search-head-") {
+					testenv.VerifyAppInstall(deployment, testenvInstance, testenvInstance.GetName(), podName, appListV1, true, "enabled", false)
+				} else {
+					testenv.VerifyAppInstall(deployment, testenvInstance, testenvInstance.GetName(), podName, appListV1, true, "disabled", false)
```

Comment:
Is this checking for apps installed on Deployer and Cluster Master as well ? For C3 we wont have anything installed on them. 

We need logic to differentiate based on `local` vs `cluster` and `pod type`

Reply:
I am checking that the app is disabled on Cluster-master and deployer. Even if the app is not installed it does show up under the `splunk display app` command as disabled

Reply:
Its a negative test ? If that's the case then can you please add a comment would be easy to understand.

Reply:
modified